Assets/SEE/Net/Network.cs

```diff
@@ -273,7 +256,30 @@ private void Awake()
         }
 
         /// <summary>
-        /// Makes sure that we have only one <see cref="Instance"/>.
+        /// Name of command-line argument for room password <see cref="RoomPassword"/>.
+        /// </summary>
+        private const string passwordArgument = "-password";
```

Comment:
Long-form command-line arguments like these are usually preceded by two dashes instead of just one. (Also applies to other constants below.)